templates/base.html

```diff
@@ -32,7 +32,15 @@
                     <ul class="nav navbar-nav">
                     </ul>
                     <ul class="nav navbar-nav navbar-right">
-                        <li><a href="{% url 'users:signup' %}">Sign up</a></li>
+						{% if user %}
```

Comment:
You should use 4 spaces and not tabs! See [CONTRIBUTING.md](https://github.com/7Pros/circuit/blob/develop/CONTRIBUTING.md)